document/CoreFormat.mdk

```diff
@@ -390,7 +390,8 @@ the post-processing tool should not alter any part of the `tool` object.
 {
     "name": "CodeScanner",                                       # see \(&sect;[#tool-name]\)
     "fullName": "CodeScanner 2.0, Developer Preview (en-US)",    # see \(&sect;[#tool-fullName]\)
-    "version": "2.0.0-beta",                                     # see \(&sect;[#tool-version]\)
+    "semanticVersion": "2.0.0-beta.1",                           # see \(&sect;[#tool-semanticVersion]\)
+    "version": "2.0 Beta 1 x86 (en-US)                           # see \(&sect;[#tool-version]\)
```

Comment:
The point of version, I thought, was to accurately reflect the literal version (and only version) provided by the tool. Your 'version' example looks like the full name (contains build flavor, locale, etc).

For a C# version, this would be the AssemblyVersion, for example (which isn't semver 2 compatible due to the Revision component).